rcore-fs/src/vfs.rs

```diff
@@ -210,6 +211,11 @@ pub trait INode: Any + Sync + Send {
         self.read_at(0, &mut buf.as_mut_slice()[..elf64_hdr_size])?;
         Ok(buf)
     }
+
+    /// Get the extension of this inode
+    fn ext(&self) -> Result<&Extension> {
```

Comment:
I suggest changing the return type of this method to `Option`. This way the user won't wonder what are the possible errors that may return from this method. I cannot see there are any other possible errors except the inode doesn't have an extension attached to it.

Reply:
Done, thanks for the review